README.md

```diff
@@ -53,6 +53,22 @@ tcld namespace update accepted-client-ca set -n <namespace> --ca-certificate-fil
 ```
 > :warning: If the update removes a certificate, any clients (tctl/workers) still using the removed certificate will fail to connect to the namespace after the update completes.
 
+#### Performing a Certificate Rollover:
+It's important to do a rollover process when updating your CA certificates. This allows your namespace to serve both CA certs for a period of time until traffic to your old cert is gone. To do this follow these steps:
+1. Create a single file that contains both your old and new CA cert pem blocks. You can do this by simply concatenating each pem block on a new line.
```

Comment:
```suggestion
1. Create a single file that contains both your old and new CA cert PEM blocks. You can do this by simply concatenating the PEM blocks on adjacent lines.
```

Reply:
small nit here: I feel that programmers might understand "new line" more than "adjacent lines"